### PR TITLE
Cleanup examples namespace

### PR DESCRIPTION
**Pull request by asalzburger**
As discussed:
* this removes the `FW` namespace and used consistently the `ActsExample` namespace
* at the same time it renames the targets in the Example folder to `ActsExampleWhatItDoes`
* and it removes the redundant `ActsUnitTestTopicTests` to be simply named `ActsUnitTestTopic`

I had also to fix one missing `std::array` include in order to make it compile.

### CODE REVIEW

**Comment by asalzburger** [comment]
The full list of executables after this change

```shell
ActsBenchmarkAnnulusBoundsBenchmark
ActsBenchmarkAtlasStepper
ActsBenchmarkBoundaryCheck
ActsBenchmarkEigenStepper
ActsBenchmarkRayFrustumBenchmark
ActsBenchmarkSolenoidField
ActsBenchmarkSurfaceIntersection
ActsExampleAdaptiveMultiVertexFinder
ActsExampleCKFTracks
ActsExampleFatrasAligned
ActsExampleFatrasDD4hep
ActsExampleFatrasGeneric
ActsExampleFatrasPayload
ActsExampleFatrasTGeo
ActsExampleGeantinoRecordingDD4hep
ActsExampleGeantinoRecordingGdml
ActsExampleGenParticleGun
ActsExampleGeometryAligned
ActsExampleGeometryDD4hep
ActsExampleGeometryEmpty
ActsExampleGeometryGeneric
ActsExampleGeometryPayload
ActsExampleGeometryTGeo
ActsExampleHelloWorld
ActsExampleIterativeVertexFinder
ActsExampleMagneticField
ActsExampleMagneticFieldAcess
ActsExampleMaterialMappingDD4hep
ActsExampleMaterialMappingGeneric
ActsExampleMaterialValidationDD4hep
ActsExampleMaterialValidationGeneric
ActsExamplePropagationAligned
ActsExamplePropagationDD4hep
ActsExamplePropagationEmpty
ActsExamplePropagationGeneric
ActsExamplePropagationPayload
ActsExamplePropagationTGeo
ActsExampleReadCsvGeneric
ActsExampleTruthTracks
ActsExampleVertexFitter
ActsExampleVertexReader
ActsExampleVertexWriter
ActsGenPythia8
ActsIntegrationTestATLSeeding
ActsIntegrationTestBVHNavigation
ActsIntegrationTestFatrasSimulation
ActsIntegrationTestInterpolatedSolenoidBField
ActsIntegrationTestPrintDataDirectory
ActsIntegrationTestPropagation
ActsTabulateEnergyLoss
ActsTutorialAMVF
ActsUnitTestAbortList
ActsUnitTestAccumulatedMaterialProperties
ActsUnitTestAccumulatedSurfaceMaterial
ActsUnitTestAccumulatedVolumeMaterial
ActsUnitTestActionList
ActsUnitTestAdaptiveMultiVertexFinder
ActsUnitTestAdaptiveMultiVertexFitter
ActsUnitTestAlignmentContext
ActsUnitTestAlignmentHelper
ActsUnitTestAnnealingUtility
ActsUnitTestAnnulusBounds
ActsUnitTestAtlasStepper
ActsUnitTestAuctioneer
ActsUnitTestAxes
ActsUnitTestBFieldMapUtils
ActsUnitTestBenchmarkTools
ActsUnitTestBinUtility
ActsUnitTestBinnedSurfaceMaterial
ActsUnitTestBinningData
ActsUnitTestBoundTrackParameters
ActsUnitTestBoundaryCheck
ActsUnitTestBoundingBox
ActsUnitTestCartesianSegmentation
ActsUnitTestClusterization
ActsUnitTestCombinatorialKalmanFilter
ActsUnitTestConeBounds
ActsUnitTestConeSurface
ActsUnitTestConeVolumeBounds
ActsUnitTestConstantBField
ActsUnitTestConstrainedStep
ActsUnitTestConvexPolygonBounds
ActsUnitTestCovarianceEngine
ActsUnitTestCuboidVolumeBounds
ActsUnitTestCuboidVolumeBuilder
ActsUnitTestCurvilinearTrackParameters
ActsUnitTestCutoutCylinderVolumeBounds
ActsUnitTestCylinderBounds
ActsUnitTestCylinderLayer
ActsUnitTestCylinderSurface
ActsUnitTestCylinderVolumeBounds
ActsUnitTestCylinderVolumeBuilder
ActsUnitTestDiamondBounds
ActsUnitTestDirectNavigator
ActsUnitTestDiscLayer
ActsUnitTestDiscSurface
ActsUnitTestDiscTrapezoidBounds
ActsUnitTestDoubleHitSpacePointBuilder
ActsUnitTestEllipseBounds
ActsUnitTestEventDataView3D
ActsUnitTestExtendable
ActsUnitTestExtent
ActsUnitTestExtrapolator
ActsUnitTestFatrasBarcode
ActsUnitTestFatrasChargeSelectors
ActsUnitTestFatrasEnergyLoss
ActsUnitTestFatrasHit
ActsUnitTestFatrasInteractor
ActsUnitTestFatrasKinematicCasts
ActsUnitTestFatrasLimitSelectors
ActsUnitTestFatrasParticle
ActsUnitTestFatrasParticleData
ActsUnitTestFatrasPdgSelectors
ActsUnitTestFatrasPhysicsList
ActsUnitTestFatrasProcess
ActsUnitTestFatrasScattering
ActsUnitTestFatrasSelectorHelpers
ActsUnitTestFiniteStateMachine
ActsUnitTestFittableType
ActsUnitTestFreeTrackParameters
ActsUnitTestFrustum
ActsUnitTestFullBilloirVertexFitter
ActsUnitTestGainMatrixSmoother
ActsUnitTestGainMatrixUpdater
ActsUnitTestGaussianGridTrackDensity
ActsUnitTestGenericApproachDescriptor
ActsUnitTestGenericCuboidVolumeBounds
ActsUnitTestGeometryHierarchyMap
ActsUnitTestGeometryHierarchyMapJsonConverter
ActsUnitTestGeometryID
ActsUnitTestGrid
ActsUnitTestGridDensityVertexFinder
ActsUnitTestHelpers
ActsUnitTestHomogeneousSurfaceMaterial
ActsUnitTestHomogeneousVolumeMaterial
ActsUnitTestImpactPointEstimator
ActsUnitTestInfiniteBounds
ActsUnitTestInteractions
ActsUnitTestInterpolatedBFieldMap
ActsUnitTestInterpolatedMaterialMap
ActsUnitTestInterpolation
ActsUnitTestIntersection
ActsUnitTestIntersectionHelper2D
ActsUnitTestIterativeVertexFinder
ActsUnitTestJacobian
ActsUnitTestJsonConversion
ActsUnitTestKalmanExtrapolator
ActsUnitTestKalmanFitter
ActsUnitTestKalmanVertexTrackUpdater
ActsUnitTestKalmanVertexUpdater
ActsUnitTestLayer
ActsUnitTestLayerCreator
ActsUnitTestLineBounds
ActsUnitTestLineSurface
ActsUnitTestLinearizedTrackFactory
ActsUnitTestLogger
ActsUnitTestLoopProtection
ActsUnitTestMPL
ActsUnitTestMagneticFieldInterfaceConsistency
ActsUnitTestMaterial
ActsUnitTestMaterialCollection
ActsUnitTestMaterialComposition
ActsUnitTestMaterialGridHelper
ActsUnitTestMaterialMapUtils
ActsUnitTestMaterialProperties
ActsUnitTestMeasurement
ActsUnitTestMeasurementHelpers
ActsUnitTestMultiIndex
ActsUnitTestMultiTrajectory
ActsUnitTestNavigationLayer
ActsUnitTestNavigator
ActsUnitTestParameterSet
ActsUnitTestPerigeeSurface
ActsUnitTestPlanarModuleStepper
ActsUnitTestPlaneLayer
ActsUnitTestPlaneSurface
ActsUnitTestPolyhedron
ActsUnitTestPrimitivesView3D
ActsUnitTestPropagator
ActsUnitTestProtoLayer
ActsUnitTestProtoLayerHelper
ActsUnitTestProtoSurfaceMaterial
ActsUnitTestProtoVolumeMaterial
ActsUnitTestRadialBounds
ActsUnitTestRay
ActsUnitTestRealQuadraticEquation
ActsUnitTestRectangleBounds
ActsUnitTestResult
ActsUnitTestSeedfinder
ActsUnitTestSimpleGeometry
ActsUnitTestSingleHitSpacePointBuilder
ActsUnitTestSolenoidBField
ActsUnitTestStepper
ActsUnitTestStraightLineStepper
ActsUnitTestStrawSurface
ActsUnitTestSurface
ActsUnitTestSurfaceArray
ActsUnitTestSurfaceArrayCreator
ActsUnitTestSurfaceBinningMatcher
ActsUnitTestSurfaceBounds
ActsUnitTestSurfaceIntersection
ActsUnitTestSurfaceLocalToGlobalRoundtrip
ActsUnitTestSurfaceMaterialMapper
ActsUnitTestSurfaceView3D
ActsUnitTestTGeoArb8Conversion
ActsUnitTestTGeoBBoxConversion
ActsUnitTestTGeoLayerBuilder
ActsUnitTestTGeoParser
ActsUnitTestTGeoPrimitivesHelper
ActsUnitTestTGeoTrd1Conversion
ActsUnitTestTGeoTrd2Conversion
ActsUnitTestTGeoTubeConversion
ActsUnitTestTrackDensityVertexFinder
ActsUnitTestTrackingGeometryClosureGeometry
ActsUnitTestTrackingGeometryCreation
ActsUnitTestTrackingGeometryGeoID
ActsUnitTestTrackingGeometryView3D
ActsUnitTestTrackingVolume
ActsUnitTestTrapezoidBounds
ActsUnitTestTrapezoidVolumeBounds
ActsUnitTestTypeTraits
ActsUnitTestUnitConversion
ActsUnitTestUnitVectors
ActsUnitTestVersion
ActsUnitTestVerticesHelper
ActsUnitTestVisualization3D
ActsUnitTestVolume
ActsUnitTestVolumeBounds
ActsUnitTestVolumeMaterialInteraction
ActsUnitTestVolumeMaterialMapper
ActsUnitTestVolumeView3D
ActsUnitTestZScanVertexFinder
```

**Comment by asalzburger** [comment]
Missed one:
`ActsGenPythia8`

**Comment by asalzburger** [comment]
And these two do not fit anywhere:

```shell
ActsTabulateEnergyLoss
ActsTutorialAMVF
```

**Comment by asalzburger** [comment]
> Missed one:
> `ActsGenPythia8`

Changes this one to `ActsExamplePythia8`, this PR is now ready to review.

File: Examples/Run/Vertexing/VertexAndTracksReaderAndFinderExample.cpp [hunk 1]
```diff
@@ -64,8 +64,9 @@ int main(int argc, char* argv[]) {
   sequencer.addReader(std::make_shared<RootVertexAndTracksReader>(
       vtxAndTracksReaderCfg, logLevel));
 
-  sequencer.addAlgorithm(std::make_shared<FWE::IterativeVertexFinderAlgorithm>(
-      vertexFindingCfg, logLevel));
+  sequencer.addAlgorithm(
+      std::make_shared<ActsExamplesE::IterativeVertexFinderAlgorithm>(
```
Comment (msmk0) [inline Examples/Run/Vertexing/VertexAndTracksReaderAndFinderExample.cpp:68]:
```suggestion
      std::make_shared<ActsExamples::IterativeVertexFinderAlgorithm>(
```

File: Examples/Run/Vertexing/VertexAndTracksReaderAndFinderExample.cpp [hunk 2]
```diff
@@ -54,7 +54,7 @@ int main(int argc, char* argv[]) {
   Acts::Vector3D bField(0., 0., 2. * Acts::units::_T);
 
   // Add the finding algorithm
-  FWE::IterativeVertexFinderAlgorithm::Config vertexFindingCfg;
+  ActsExamplesE::IterativeVertexFinderAlgorithm::Config vertexFindingCfg;
```
Comment (msmk0) [inline Examples/Run/Vertexing/VertexAndTracksReaderAndFinderExample.cpp:57]:
```suggestion
  ActsExamples::IterativeVertexFinderAlgorithm::Config vertexFindingCfg;
```

File: Examples/Run/Vertexing/AdaptiveMultiVertexFinderExample.cpp [hunk 3]
```diff
@@ -87,10 +87,10 @@ int main(int argc, char* argv[]) {
       std::make_shared<TrackSelector>(selectorConfig, logLevel));
 
   // Add the finding algorithm
-  FWE::AdaptiveMultiVertexFinderAlgorithm::Config vertexFindingCfg;
+  ActsExamplesE::AdaptiveMultiVertexFinderAlgorithm::Config vertexFindingCfg;
```
Comment (msmk0) [inline Examples/Run/Vertexing/AdaptiveMultiVertexFinderExample.cpp:90]:
```suggestion
  ActsExamples::AdaptiveMultiVertexFinderAlgorithm::Config vertexFindingCfg;
```

File: Examples/Run/Vertexing/AdaptiveMultiVertexFinderExample.cpp [hunk 4]
```diff
@@ -87,10 +87,10 @@ int main(int argc, char* argv[]) {
...
   vertexFindingCfg.trackCollection = selectorConfig.output;
   sequencer.addAlgorithm(
-      std::make_shared<FWE::AdaptiveMultiVertexFinderAlgorithm>(
+      std::make_shared<ActsExamplesE::AdaptiveMultiVertexFinderAlgorithm>(
```
Comment (msmk0) [inline Examples/Run/Vertexing/AdaptiveMultiVertexFinderExample.cpp:93]:
```suggestion
      std::make_shared<ActsExamples::AdaptiveMultiVertexFinderAlgorithm>(
```

File: Examples/Run/Vertexing/IterativeVertexFinderExample.cpp [hunk 5]
```diff
@@ -87,11 +87,12 @@ int main(int argc, char* argv[]) {
       std::make_shared<TrackSelector>(selectorConfig, logLevel));
 
   // Add the finding algorithm
-  FWE::IterativeVertexFinderAlgorithm::Config vertexFindingCfg;
+  ActsExamplesE::IterativeVertexFinderAlgorithm::Config vertexFindingCfg;
```
Comment (msmk0) [inline Examples/Run/Vertexing/IterativeVertexFinderExample.cpp:90]:
```suggestion
  ActsExamples::IterativeVertexFinderAlgorithm::Config vertexFindingCfg;
```

File: Examples/Run/Vertexing/IterativeVertexFinderExample.cpp [hunk 6]
```diff
@@ -87,11 +87,12 @@ int main(int argc, char* argv[]) {
...
   vertexFindingCfg.trackCollection = selectorConfig.output;
   vertexFindingCfg.bField = trkConvConfig.bField;
-  sequencer.addAlgorithm(std::make_shared<FWE::IterativeVertexFinderAlgorithm>(
-      vertexFindingCfg, logLevel));
+  sequencer.addAlgorithm(
+      std::make_shared<ActsExamplesE::IterativeVertexFinderAlgorithm>(
```
Comment (msmk0) [inline Examples/Run/Vertexing/IterativeVertexFinderExample.cpp:94]:
```suggestion
      std::make_shared<ActsExamples::IterativeVertexFinderAlgorithm>(
```

File: Examples/Run/Vertexing/TutorialAMVFExample.cpp [hunk 7]
```diff
@@ -87,10 +87,10 @@ int main(int argc, char* argv[]) {
       std::make_shared<TrackSelector>(selectorConfig, logLevel));
 
   // Add the finding algorithm
-  FWE::TutorialAMVFAlgorithm::Config vertexFindingCfg;
+  ActsExamplesE::TutorialAMVFAlgorithm::Config vertexFindingCfg;
```
Comment (msmk0) [inline Examples/Run/Vertexing/TutorialAMVFExample.cpp:90]:
```suggestion
  ActsExamples::TutorialAMVFAlgorithm::Config vertexFindingCfg;
```

File: Examples/Run/Vertexing/TutorialAMVFExample.cpp [hunk 8]
```diff
@@ -87,10 +87,10 @@ int main(int argc, char* argv[]) {
...
   vertexFindingCfg.trackCollection = selectorConfig.output;
-  sequencer.addAlgorithm(
-      std::make_shared<FWE::TutorialAMVFAlgorithm>(vertexFindingCfg, logLevel));
+  sequencer.addAlgorithm(std::make_shared<ActsExamplesE::TutorialAMVFAlgorithm>(
```
Comment (msmk0) [inline Examples/Run/Vertexing/TutorialAMVFExample.cpp:92]:
```suggestion
  sequencer.addAlgorithm(std::make_shared<ActsExamples::TutorialAMVFAlgorithm>(
```

**Comment by msmk0** [comment]
@asalzburger Can you update to the current master branch. I will merge it afterwards.

**Pull request merged**